modules/claims_api/app/models/claims_api/auto_established_claim.rb

```diff
@@ -99,6 +100,51 @@ def uploader
 
     private
 
+    EVSS_TZ = 'Central Time (US & Canada)'
+
+    def recent_service_periods_end_dates
+      end_dates = form_data.dig('serviceInformation', 'servicePeriods').map do |service_period|
+        unless service_period['serviceBranch'].include?('Reserve') ||
+               service_period['serviceBranch'].include?('National Guard')
+          service_period['activeDutyEndDate']
+        end
+      end
+
+      end_dates.compact
+    end
+
+    def user_supplied_rad_date
+      end_dates = recent_service_periods_end_dates
+      end_dates << form_data.dig('serviceInformation',
+                                 'reservesNationalGuardService',
+                                 'title10Activation',
+                                 'anticipatedSeparationDate')
+      end_dates.compact!
+      return nil if end_dates.blank?
+
+      end_dates.max.in_time_zone(EVSS_TZ).to_date
+    end
+
+    def days_until_release
+      return 0 if user_supplied_rad_date.blank?
+
+      form_submission_date = created_at.presence || Time.now.in_time_zone(EVSS_TZ)
+      @days_until_release ||= user_supplied_rad_date - form_submission_date.to_date
+    end
+
+    def bdd_qualified?
+      if days_until_release > 180
+        return false if (recent_service_periods_end_dates - [user_supplied_rad_date.to_s]).any?
+
+        raise ::Common::Exceptions::UnprocessableEntity.new(
+          detail: 'User may not submit BDD more than 180 days prior to RAD date',
+          source: 'DataTranslationAllClaim'
```

Comment:
I think this might be a copy/paste leftover.

Reply:
fixed: https://github.com/department-of-veterans-affairs/vets-api/pull/7624/commits/e5e4214aba044160362b2b16829374b87ae76efa